tss/rsa/keyshare.go

```diff
@@ -21,6 +32,13 @@ type KeyShare struct {
 
 	Players   uint
 	Threshold uint
+
+	// It stores keys to produce verifiable signature shares.
+	// If it's nil, signature shares are still produced but
+	// they are not verifiable.
+	// This field is present only if the RSA private key is
+	// composed of two safe primes.
+	vk *VerifyKeys
```

Comment:
This should be added to the MarshalBinary/UnmarshalBinary functions https://github.com/cloudflare/circl/blob/tssRSAverif/tss/rsa/keyshare.go#L51

Reply:
to ease review, I will propose changes to the serialization functions in a different PR.